theme/js/dataset-recommendations.js

```diff
@@ -0,0 +1,67 @@
+const recoUrl = 'https://raw.githubusercontent.com/etalab/datasets_reco/master/reco_json/';
+const maxRecos = 2;
+
+function getDatasetId() {
+    const selector = '#json_ld';
+    const dataset = JSON.parse(document.querySelector(selector).text);
+    return dataset && dataset['@id'];
+}
```

Comment:
Maybe expose the dataset object from https://github.com/opendatateam/udata/blob/master/js/front/dataset/index.js#L60-L69
et reuse it here (avoid dual JSON parsing) (Not mandatory, for further improvements)

Reply:
Ok for further improvement.